package.json

```diff
@@ -77,7 +88,8 @@
     "build": "webpack -p",
     "build:analyze": "webpack -p --env.analyze",
     "test": "karma start",
-    "lint": "eslint '@(app|test)/**/*.js'",
+    "lint": "eslint '@(app|test)/**/*.{js,ts,tsx}'",
+    "lint:ts": "tsc",
```

Comment:
You should make Travis run this. (Or Github Actions hehe)

Reply:
Ah yep! Github actions eventually 🙂